docs/demo/kitchen-sink/web-components.rst

```diff
@@ -0,0 +1,104 @@
+.. INSPIRED FROM sphinx-design documentation
+
+Web components
+==============
+
+Cards and tabs are a key functionalities in many documentations depending on pydata-sphinx-theme. Both `sphinx-design <https://sphinx-design.readthedocs.io/en/latest/index.html>`__ and `sphinx-panels <https://sphinx-panels.readthedocs.io/en/latest/>`__ can be used with this theme. We overwrote some of their display features to make them compatible with our supported themes. For usage, please refer to their documentations.
+
+.. danger::
+
+    To use the :code:`sphinx-panels` extention, you need to integrate theses lines to overwrite the shadows of the panels in your custom :code:`.css` file:
+
+    .. code-block:: css
+
+        /* overwrite panels shadows using pydata-sphinx-theme variable */
+        .shadow {
+            box-shadow: 0 0.5rem 1rem var(--pst-color-shadow) !important;
+        }
+
+    This modification is not needed when using the :code:`sphinx-design` extention.
+
+Here you can find some examples of the **cards** and **tabs** created with the :code:`sphinx-design` extention.
+
+Cards
+-----
+
+.. grid::
+
+    .. grid-item-card:: Content of the first card
+
+    .. grid-item-card:: Content of the second card
+
+        :bdg-primary:`example`
+
+    .. grid-item-card::
+
+        .. dropdown:: :fa:`eye,mr-1` third card
```

Comment:
There seems to be an awkward "double outline" effect on this card, can we figure out how to remove that?